pkg/cmd/errors/login.go

```diff
@@ -0,0 +1,49 @@
+package errors
```

Comment:
> I think that makes sense, @liggitt or @deads2k do you have an opinion on this?

If you're going to use this type to collapse the `pkg/bootstrap/docker` usage, then it can live here.  If its only call site is `pkg/cmd/cli/cmd`, then it should live there.